packages/core/src/js/feedback/FeedbackForm.tsx

```diff
@@ -0,0 +1,241 @@
+import type { SendFeedbackParams } from '@sentry/core';
+import { captureFeedback, getCurrentScope, lastEventId, logger } from '@sentry/core';
+import * as React from 'react';
+import type { KeyboardTypeOptions } from 'react-native';
+import {
+  Alert,
+  Image,
+  Keyboard,
+  KeyboardAvoidingView,
+  SafeAreaView,
+  ScrollView,
+  Text,
+  TextInput,
+  TouchableOpacity,
+  TouchableWithoutFeedback,
+  View
+} from 'react-native';
+
+import { sentryLogo } from './branding';
+import { defaultConfiguration } from './defaults';
+import defaultStyles from './FeedbackForm.styles';
+import type { FeedbackFormProps, FeedbackFormState, FeedbackFormStyles,FeedbackGeneralConfiguration, FeedbackTextConfiguration, Navigation } from './FeedbackForm.types';
+import { feedbackIcon }  from './icons';
+
+let feedbackFormHandler: (() => void) | null = null;
+
+const setFeedbackFormHandler = (handler: () => void): void => {
+  feedbackFormHandler = handler;
+};
+
+const clearFeedbackFormHandler = (): void => {
+  feedbackFormHandler = null;
+};
+
+/**
+ * @beta
+ * Shows the feedback form screen that sends feedback to Sentry using Sentry.captureFeedback.
+ * @param navigation The navigation object from React Navigation
+ */
+export const showFeedbackForm = (navigation: Navigation): void => {
```

Comment:
I think we should avoid depending on specific navigation lib and enforcing specific screen name.

I imagine the auto injection to work when uset used `Sentry.wrap(AppRoot)` and then anywhere `Sentry.showFeedbackForm()` is called the form will show up (we can ensure using style it's on to of other content, it won't show as navigation, but I think that's acceptable).

But also I'm open to discuss the approach. 

This will require to store the feedback options in the `Sentry.init(options)`